src/main/java/jenkins/scm/api/SCMSourceOwners.java

```diff
@@ -69,7 +70,7 @@
         public Iterator<SCMSourceOwner> iterator() {
             Jenkins j = Jenkins.getInstance(); // TODO 1.590+ getActiveInstance
```

Comment:
@andresrc 
Answer the following questions please:

* Is this code running in a plugin?
* Is this code only ever running in the same JVM as the Jenkins singleton?

If the answer to the above two questions are both yes (and let's be fair here, the first one has to be a yes) then `Jenkins.getInstance()` will **never** be `null` (or if it is `null` then that is a bug in Jenkins core)

So either 
* this is a class that is not intended to be called from slave agent JVMs, in which case you should use `Jenkins.getActiveInstance()` or 
* this is a class that is intended to be called from slave agent JVMs, in which case go away and rewrite this not to use the `Jenkins` class as we should not be loading the `Jenkins` class into slave agent JVMs...

Once you unpin that logic bomb, you will most likely be just using `Jenkins.getActiveInstance()` ;-)

I am expecting a follow-up PR